### ISSUE TITLE
Handling exceptions in 6.0

### ISSUE THREAD

**Issue by BoShurik** [8p/GuzzleBundle#117]
This code
```
try {
    $httpResponse = $this->httpClient->sendRequest($httpRequest);
} catch (HttpException $exception) {
    throw ExceptionHandler::handle($exception);
}
```
no longer catch an 401 exception (may be other exceptions too)
The problem is in this [commit](https://github.com/8p/GuzzleBundle/commit/a8bc12640aa0c17bc0a69b4833151bf5c616c7ed) by @thomasjohansen

**Comment by florianpreusner** [comment]
Thanks for your report. 5.4 was removed a couple of days ago. Are you sure that you are using this version? We have this "feature" in v6.0.
So maybe the title should be changed.

**Comment by BoShurik** [comment]
Just update to 5.4 version
https://packagist.org/packages/eightpoints/guzzle-bundle

**Comment by BoShurik** [comment]
In 6.0 the problem still exists

**Comment by florianpreusner** [comment]
Mhhh, I removed this version from packagist, but when clicking update it will be reactivated, strange. Sorry for the trouble. I will take a look into this later today.

**Comment by BoShurik** [comment]
@florianpreusner looks like you must remove it from releases

**Comment by florianpreusner** [comment]
@BoShurik I can' reproduce your issue. "sendRequest" is a method provided by your own class, or? Inside you are using \GuzzleHttp\Client->request()?

When I'm getting a 401 I get the exception (GuzzleHttp\Exception\ClientException) and can catch it. That works with the modification by @thomasjohansen and also without. So I can't see a change in the behavior. Maybe you should adjust your class that will be catched. I can't see a place where "HttpException" will be thrown.

```php
$client = $this->get('guzzle.client.test_client');        
try {
    $response = $client->request('GET', 'https://httpbin.org/status/401');
} catch(\GuzzleHttp\Exception\ClientException $e) {
    // here we go
}
```

**Comment by BoShurik** [comment]
Forgot to mention that I use `httplug` and [php-http/guzzle6-adapter](https://github.com/php-http/guzzle6-adapter/blob/master/src/Promise.php#L48)
`HttpException`  is `Http\Client\Exception\HttpException`

**Comment by TheLevti** [comment]
Can it be that my latest PR #115 fixed this issue (see #114)? I had the same problem of not getting an exception.

**Comment by BoShurik** [comment]
It works! Can you add tag?

**Comment by florianpreusner** [comment]
Just released v6.0.1
https://github.com/8p/GuzzleBundle/releases/tag/v6.0.1
